src/Core/Addon.php

```diff
@@ -226,38 +199,33 @@ public static function install($addon)
 	 */
 	public static function reload()
 	{
-		$addons = DI::config()->get('system', 'addon');
-		if (strlen($addons)) {
-			$r = DBA::select('addon', [], ['installed' => 1]);
-			if (DBA::isResult($r)) {
-				$installed = DBA::toArray($r);
-			} else {
-				$installed = [];
-			}
+		$installed = DBA::selectToArray('addon', [], ['installed' => 1]);
+
+		$addon_list = [];
+		foreach ($installed as $addon) {
+			$addon_list[] = $addon['name'];
```

Comment:
If `$addon_list` isn’t used for anything else, you can loop on `$installed` directly.